docs/apache-airflow/core-concepts/params.rst

```diff
@@ -124,8 +124,10 @@ Another way to access your param is via a task's ``context`` kwarg.
         print(context["params"]["my_int_param"])
 
     PythonOperator(
-        task_id="print_my_int_param",
-        python_callable=print_my_int_param,
+      task_id="print_my_int_param",
+      python_callable=print_x,
+      provide_context=True,
```

Comment:
```suggestion
```

You don't have to set `provide_context` since [Airflow 2.0
](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-operators-python-pythonoperator)

Reply:
Got it. Will fix.